web/src/app/administration/components/classification-list/classification-list.component.html

```diff
@@ -53,7 +53,7 @@
 	<!-- CLASSIFICATION TREE -->
 	<taskana-spinner class="col-xs-12" [isRunning]="requestInProgress" positionClass="centered-spinner-whole-screen"></taskana-spinner>
 	<taskana-tree class="col-xs-12" *ngIf="(classifications && classifications.length) else empty_classifications"
-	 [filterText]="inputValue" [filterIcon]="selectedCategory"></taskana-tree>
+	 [filterText]="inputValue" [filterIcon]="selectedCategory" (switchTaskanaSpinnerEmit)="setRequestInProgress($event)"></taskana-tree>
```

Comment:
Would a simple requestInProgress=$event not suffice?

Reply:
done